coc.md

```diff
@@ -21,15 +21,18 @@ title: Code of conduct
 
 <p><b>Contact Information</b></p>
 
-<p>If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference staff. Conference staff will be wearing “PyConf Hyderabad Staff” badges.
-You may also ask to be put in touch with the conference chair - Manivannan Elangovan.</p>
+<p>If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference volunteer. Conference volunteer will be wearing “PyConf Hyderabad Volunteer” badges.
```

Comment:
<p>If you are being harassed, or notice that someone else is being harassed, or have any other concerns, please contact any of the conference volunteer. Conference volunteer will be wearing “PyConf Hyderabad Volunteer” badges.